src/lib/indexer/eventProcessor.ts

```diff
@@ -1,10 +1,10 @@
 import { v4 as UUIDv4 } from 'uuid'
 
 import { UUID } from '../../models/strings'
-import { TransactionRow } from '../db'
-import { ChangeSet, CertificateRecord } from './changeSet'
+import { TransactionRow } from '../../lib/db/types'
```

Comment:
DB changes as separate PR from API controller, testing first with initiation and issuance?

Reply:
I think this is fine. The DB changes look pretty clean